### PR TITLE
Bring back FUSE

### PR DESCRIPTION
**Pull request by blankdots**
Bringing FUSE back, as agreed.
However we are going to keep both solutions for the inbox Apache Mina and FUSE.
* Documentation updated to reflect both solutions
* switch added in the bootstrap and the `DEFAULT_INBOX` is fuse (decision is based on first solution first served).
* `ramfs` brought back for FUSE, because it does not hurt
* the flag can be used as `make bootstrap ARGS="--inbox mina"` used in Travis.
* Travis only tests Apache Mina Inbox right now as we need to have specific configuration for FUSE namely these two parameters
  - `images.name.inbox = nbisweden/ega-inbox`
  - `inbox.folder.path = /lega`

> How should we select which should be the DEFAULT option?

As mentioned I propose first implemented first selected, but we can also vote.

> Should we have Integration tests in Travis for FUSE as well?

We should, although it will take longer and we have to build different stages.

> Did you make typos in multiple commits ?

Yes I did, and i deeply apologise for that, and I hope it will not happen, but it is not a promise.

### CODE REVIEW

**Comment by dtitov** [comment]
Good job 👍 

Regarding the default inbox: I, of course, suggest voting. As for me, the Unix-based inbox is now more of a legacy inbox that we agreed to keep just to make all parties happy, thus it shouldn't be the default one. But it's only my feeling, we can discuss it on Monday together.

File: deployments/docker/bootstrap/lega.sh [hunk 1]
```diff
@@ -329,6 +374,7 @@ volumes:
   s3:
 EOF
 
+sed $'s/\t/  /' ${PRIVATE}/lega.yml >>/dev/null
```
Comment (silverdaz) [inline deployments/docker/bootstrap/lega.sh]: Eh? You correct a file that you, yourself, produce?

**Review by silverdaz — APPROVED** [review]
Looks good to me.
It feels like the files have been (re)copied.
I may be wrong and it's just the interface giving me that feeling.
I don't think it is the way to bring back the files. I'd rather go for a merge, to keep the history. We should as Johan, he'll know.

File: deployments/docker/bootstrap/lega.sh (hunk 1 above)
Comment (blankdots) [reply deployments/docker/bootstrap/lega.sh]: i forgot that there, had some issues with indentation, turned out to be from the editor - I will remove

**Comment by blankdots** [comment]
I should have added this to the list of expected questions, about the file (re)copying.
Yes I did choose the simple route and restored the file, which can be viewed as (re)copying, as we did several changes after those files were deleted.

**Pull request merged**